app/com/lynxanalytics/biggraph/frontend_operations/WorkflowOperations.scala

```diff
@@ -609,8 +609,14 @@ class WorkflowOperations(env: SparkFreeEnvironment) extends ProjectOperations(en
         input.kind match {
           case BoxOutputKind.Project =>
             val project = projectInput("graph")
-            PythonUtilities.derive(params("code"), pythonInputs, pythonOutputs, project)
-            Map(context.box.output("graph") -> BoxOutputState.from(project))
+            if (params("outputs") == "plot") {
```

Comment:
So you indicate that you want a plot by just declaring the output as `plot`. I think that's not too bad, since the normal output declarations will always be of the `name:type` form, so there's no confusion. But still, maybe `plot` looks too plain? Let me know if you have a better idea. I didn't want a whole new box, because they are quite similar otherwise.